dhcpv6/nclient6/client.go

```diff
@@ -290,9 +290,17 @@ type Matcher func(*dhcpv6.Message) bool
 // IsMessageType returns a matcher that checks for the message type.
 //
 // If t is MessageTypeNone, all packets are matched.
```

Comment:
Why does it match if MessageType is None?

Reply:
yeah you're right i'm not sure that made sense